lib/biokbase/catalog/controller.py

```diff
@@ -721,7 +671,7 @@ def list_local_functions(self, params):
         else:
             release_tag = 'release'
         if 'release_tag' in params:
-            if not isinstance(params['release_tag'],basestring):
+            if not isinstance(params['release_tag'],str):
```

Comment:
Are these changes from 'basestring' to 'str' important?

Reply:
basestring is no more in Python 3